easybuild/easyconfigs/v/vsearch/vsearch-2.4.3-foss-2016b.eb

```diff
@@ -0,0 +1,42 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'ConfigureMake'
+
+name = 'vsearch'
+version = '2.4.3'
+
+homepage = '://github.com/torognes/vsearch'
+description = """We have implemented a tool called VSEARCH which supports de novo and reference based
+ chimera detection, clustering, full-length and prefix dereplication, rereplication, reverse
+ complementation, masking, all-vs-all pairwise global alignment, exact and global alignment searching,
+ shuffling, subsampling and sorting. It also supports FASTQ file analysis, filtering, conversion and
+ merging of paired-end reads.
+
+ VSEARCH stands for vectorized search, as the tool takes advantage of parallelism in the form of SIMD
+ vectorization as well as multiple threads to perform accurate alignments at high speed. VSEARCH uses
+ an optimal global aligner (full dynamic programming Needleman-Wunsch), in contrast to USEARCH which
+ by default uses a heuristic seed and extend aligner. This usually results in more accurate alignments
+ and overall improved sensitivity (recall) with VSEARCH, especially for alignments with gaps."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+sources = ['v%(version)s.tar.gz']
+source_urls = ['https://github.com/torognes/vsearch/archive/']
+
+builddependencies = [('Autotools', '20150215')]
+
+dependencies = [
+    ('zlib', '1.2.8'),
+    ('bzip2', '1.0.6'),
+]
+
+preconfigopts = ' ./autogen.sh && '
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ["."]
```

Comment:
please fix to a proper sanity check rather than recycling this hack invented by @fgeorgatos ;)